cybergis/general/summa.py

```diff
@@ -152,3 +173,103 @@ def download(self):
         self.connection.download(os.path.join(self.remote_model_folder_path, "output"),
                                  self.local_job_folder_path, remote_is_folder=True)
         self.connection.download(self.remote_slurm_out_file_path, self.local_job_folder_path)
+
+
+
+
+class SummaCometJob(KeelingJob):
```

Comment:
@CarnivalBug Same copy and paste issue? Can we directly inherit SummaKeelingJob to avoid copy and paste?
something like?
class SummaCometJob(SummaKeelingJob):
    pass